scripts/mypy.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e
+echo 'Starting mypy checks'
+mypy allennlp --ignore-missing-imports
```

Comment:
What imports are missing that it needs to ignore?  I guess pylint checks for this, too, so it's not really a correctness issue, I'm just surprised it's necessary.